inc/api/endpoints/controller/class-customtype.php

```diff
@@ -65,6 +65,11 @@ public function get_item_schema(): array {
 				],
 			],
 		];
+		$schema['properties']['status'] = [
+			'description' => __( 'The status for the post.' ),
+			'type'        => 'string',
+			'context' => [ 'view', 'edit', 'embed' ],
+		];
```

Comment:
Expose the `status` of the content using the API -- this allows us to know if the content is `publish` or `draft` for example.